Demo/EarlGreyExample/Podfile

```diff
@@ -39,7 +39,7 @@ target TEST_TARGET_SWIFT do
 end
 
 post_install do |installer|
-  load('configure_earlgrey_pods.rb')
```

Comment:
Yes. And I'm suggesting that you don't. We could chat on Slack about this as well. But this is the structure I would've used here - 

EarlGreyExample - 

  * EarlGreyExampleTests target - uses the configure_earlgrey_pods.rb script.
  * EarlGreyExampleGemTests target - uses the earlgrey gem.
  * EarlGreyExampleSwiftTests target - can use either.

The point being to demonstrate that either / both ways can be be used for starting up EarlGrey. From what I'm seeing here, we're removing the configure_earlgrey_pods.rb script altogether, when I feel we should wait out until the next release or so to do that.